### PR TITLE
Updated Order API documentation about Refund

### PR DESCRIPTION
**Pull request by anzebra**
Added documentation about return + refund

### CODE REVIEW

File: source/reference/stable/order-api/create-return.md [hunk 1]
```diff
@@ -59,7 +59,28 @@ This will create return for given shipment and items. Return will be created eve
           :required: false
 
      - Insert returned items into stock. If not provided ``Default stock action on return`` setting on the store will be used.
+   
+   * - ``handlingCost``
 
+       .. type:: float
```
Comment (fransr) [inline source/reference/stable/order-api/create-return.md]: Can also be a bool, right?

File: source/reference/stable/order-api/create-return.md [hunk 2]
```diff
@@ -59,7 +59,28 @@ This will create return for given shipment and items. Return will be created eve
...
+          :required: false
+
+     - Handling cost.
```
Comment (fransr) [inline source/reference/stable/order-api/create-return.md]: "If the handling cost of the order should be included in the return or not. Supports either a value, or a boolean to specify it it should be included"

File: source/reference/stable/order-api/create-return.md [hunk 3]
```diff
@@ -59,7 +59,28 @@ This will create return for given shipment and items. Return will be created eve
...
+
+   * - ``shippingCost``
+
+       .. type:: float | boolean
+          :required: false
+
+     - Shipping cost. If set to ``true`` returns full shipping amount.
```
Comment (fransr) [inline source/reference/stable/order-api/create-return.md]: If set to `true` the complete shipping cost will be included in the return.

File: source/reference/stable/order-api/create-return.md [hunk 4]
```diff
@@ -59,7 +59,28 @@ This will create return for given shipment and items. Return will be created eve
...
+   
+   * - ``returnCost``
+
+       .. type:: float
+          :required: false
+
+     - Return cost.
```
Comment (fransr) [inline source/reference/stable/order-api/create-return.md]: If the return itself should have a cost to it. Providing a value here will deduct the amount of money to refund.

File: source/reference/stable/order-api/create-return.md [hunk 5]
```diff
@@ -73,6 +94,33 @@ This will create return for given shipment and items. Return will be created eve
...
+
+       .. list-table::
+          :widths: auto
+
+          * - ``refundPayment``
+
+              .. type:: boolean
+                 :required: false
+
+            - Set the flag to ``true`` to refund return.
```
Comment (fransr) [inline source/reference/stable/order-api/create-return.md]: If this is set to `true` the payment provider will get a refund request to pay back the money to the customer. The amount that will be refunded is the amount for the products, the handling, shipping and return costs.

File: source/reference/stable/order-api/create-return.md [hunk 6]
```diff
@@ -59,7 +59,59 @@ This will create return for given shipment and items. Return will be created eve
...
+
+     - Shipping cost.
+
+   * - ``shippingCostFromShipment``
+
+       .. type:: boolean
+          :required: false
+          :default: false
+
+     - Set to `true` to use Shipping cost from Shipment.
```
Comment (arius86) [inline source/reference/stable/order-api/create-return.md]: You need to provide _shippingCost_ value if you choose to use this option.

File: source/reference/stable/order-api/create-return.md [hunk 7]
```diff
@@ -59,7 +59,59 @@ This will create return for given shipment and items. Return will be created eve
...
+
+     - Voucher value, should always be positive value.
+
+   * - ``voucherValueFromShipment``
+
+       .. type:: boolean
+          :required: false
+          :default: false
+
+     - Set to `true` to use Voucher value from Shipment.
```
Comment (arius86) [inline source/reference/stable/order-api/create-return.md]: You need to provide _voucherValue_ if you choose to use this option.

File: source/reference/stable/order-api/create-return.md [hunk 8]
```diff
@@ -59,7 +59,59 @@ This will create return for given shipment and items. Return will be created eve
...
+
+     - Handling cost.
+
+   * - ``handlingCostFromShipment``
+
+       .. type:: boolean
+          :required: false
+          :default: false
+
+     - Set to `true` to use Handling cost from Shipment.
```
Comment (arius86) [inline source/reference/stable/order-api/create-return.md]: You need to provide _handlingCost_ value if you choose to use this option.

File: source/reference/stable/order-api/create-return.md (hunk 7 above)
Comment (fransr) [reply source/reference/stable/order-api/create-return.md]: You cannot provide `voucherValue` at the same time this option is used. You mean like that @arius86? :)

Comment (arius86) [reply source/reference/stable/order-api/create-return.md]: Haha yes 😂

**Pull request merged**